docs/guide/data_migration/importing_data.md

```diff
@@ -43,6 +43,53 @@ The following modes are available for specific objects:
 | `segment`            | &#10004; | &#10004; | &#10004; |
 | `setting`            | &#10004; | &#10004; | &#10004; |
 
+### Repeatable steps
+
+You can run the same migration step multiple times by using the special `repeatable` migration type.
+
+A repeatable migration performs the defined migration steps as many times as the `iterations` setting declares.
+
+``` yaml hl_lines="4"
+[[= include_file('code_samples/data_migration/examples/repeatable_step.yaml', 0, 5) =]]
+```
+
+!!! tip
+
+    You can use repeatable migration steps, for example,
+    to quickly generate large numbers of Content items for testing purposes.
+
+You can vary the operations using the iteration counter.
+
+For example, to create five Folders, with names ranging from "Folder 0" to "Folder 4", you can run the following migration using the iteration counter `i`:
+
+``` yaml hl_lines="16"
+[[= include_file('code_samples/data_migration/examples/repeatable_step.yaml', 0, 16) =]]
+```
+
+To vary the content name, the migration above uses Symfony expression syntax.
+The expression is enclosed in `###` and any repeated string, in this case, `SSS`.
```

Comment:
I think we need to find some better words to explain this :confused: 

At the very least, we should show a "template" of sorts: `###<IDENTIFIER> <EXPRESSION> <IDENTIFIER>###`.

We can look to [PHP heredoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) documentation for inspiration. Syntax is similar there.

Additionaly, after the changes that introduced the RepeatableStep, expression syntax can be also used in each & every other migration. Repeatable step is an instance where it's almost necessary to use it, since rarely we can (or make sense) to execute the same exact migration twice, without any differences.